vector/src/main/res/values/strings.xml

```diff
@@ -724,7 +724,7 @@
     <!-- Note to translators: the translation MUST contain the string "${app_name}", which will be replaced by the application name -->
     <string name="template_settings_call_ringtone_use_app_ringtone">Use default ${app_name} ringtone for incoming calls</string>
     <string name="settings_call_ringtone_use_default_stun">Allow fallback call assist server</string>
-    <string name="settings_call_ringtone_use_default_stun_sum">Will use "%s" as assist when your homeserver does not offer one (your IP address will be shared during a call)</string>
+    <string name="settings_call_ringtone_use_default_stun_sum">Will use %s as assist when your homeserver does not offer one (your IP address will be shared during a call)</string>
```

Comment:
I think this copy is missleading. IP will be shared with who? the peer yes, the stun yes.
Not using it will still share your IP with the peer.
So maybe your IP will be seen by the stun server? (not sure how valuable/actionable this information is for the user, given that not doing it will make calls fails depending on your network)